docs/src/components/Toc.js

```diff
@@ -49,23 +49,27 @@ function throttle(func, wait) {
 }
 
 function useThrottledOnScroll(callback, delay) {
-  const throttledCallback = React.useMemo(() => throttle(callback, delay), [callback, delay]);
+  const throttledCallback = useMemo(() => throttle(callback, delay), [callback, delay]);
 
-  React.useEffect(() => {
+  useEffect(() => {
     window.addEventListener('scroll', throttledCallback);
     return () => {
       window.removeEventListener('scroll', throttledCallback);
     };
   }, [throttledCallback]);
 }
 
-export default function Toc({ cards }: {| cards: Array<Node> |}): Node {
+type Props = {|
```

Comment:
No material changes, just adapting to our typical component conventions

Reply:
ditto. good cleaning